### PR TITLE
Remove ospaths

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Removed `ospaths` module, was deprecated since `0.19`, the `os` module has all the functionality.
- Was a `import` / `export` of `os` anyways.

### Third party packages failing

- [x] https://github.com/unicredit/nimcuda/blob/a562a4aeeb74bab00a7d03353f9bc4636258b197/nimcuda.nimble#L13 :arrow_right: https://github.com/unicredit/nimcuda/pull/9#issue-443818119
- [x] https://github.com/brentp/nim-plotly/blob/master/plotly.nimble#L15 :arrow_right: https://github.com/brentp/nim-plotly/pull/62#issue-443817372
- [ ] https://github.com/vegansk/nimfp/blob/master/config.nims#L1
- [x] https://github.com/NimParsers/parsetoml/blob/master/parsetoml.nimble#L14 :arrow_right: https://github.com/NimParsers/parsetoml/pull/37#issue-443816481
- [x] https://github.com/tulayang/asyncmysql/blob/master/config.nims#L1
- [x] https://github.com/status-im/nim-testutils/blob/master/testutils/fuzzing_engines.nim#L1 :arrow_right: https://github.com/status-im/nim-testutils/pull/26#issue-443814146
- [x] https://github.com/Vindaar/ggplotnim/blob/master/ggplotnim.nimble#L42 :arrow_right: https://github.com/Vindaar/ggplotnim/pull/86#issue-443815559

Total ~~`7`~~ `1`

### CODE REVIEW

**Comment by ghost** [comment]
I think there are still libraries/code depending on it :(
https://github.com/search?o=desc&q=language%3Anim+ospaths&s=indexed&type=Code

**Comment by juancarlospaco** [comment]
Everything is on `os`, then you can just pass `--import:os` and it will compile.

```console
$ echo "import ospaths" > file.nim
$ touch ospaths.nim
$ nim c --import:os file.nim
CC: stdlib_os.nim
CC: file.nim
Hint:  [Link]
Hint: 43729 lines; 1.211s; 61.059MiB peakmem; Debug build; proj: file.nim; out: file [SuccessX]

$
```
:shrug:

**Comment by juancarlospaco** [comment]
@Araq Ready to merge.

File: changelog.md [hunk 1]
```diff
@@ -18,6 +18,105 @@
 
 ## Language changes
 
+- The `=destroy` hook no longer has to reset its target, as the compiler now automatically inserts
```
Comment (ringabout) [inline changelog.md:21]: Please remove all of  wrong items in changelog.

**Comment by Araq** [comment]
So by removing a deprecated module we break important packages? Come on, so leave it in.

**Pull request closed without merging**